eng/pipelines/templates/jobs/ci.yml

```diff
@@ -209,6 +209,10 @@ jobs:
         inputs:
           ignoreDirectories: '.nuget'
 
+      - pwsh: |
+          Copy-Item -Path '$(Build.SourcesDirectory)/eng/CredScanSuppression.json' -Destination `$(Build.ArtifactStagingDirectory)/CredScanSuppression.json` -Force
```

Comment:
Why is this needed? I would like to avoid this.

Reply:
removed